.lintstagedrc.json

```diff
@@ -1,3 +1,3 @@
 {
-  "*.{ts,tsx}": ["npm run lint", "npm run format"]
+  "*.{ts,tsx}": ["npm run lint:check", "npm run format:fix", "npm run format:check"]
```

Comment:
I think we can remove the `npm run format:fix` now that we have the `npm run format:check`.